BatchCave.py

```diff
@@ -2114,41 +2119,8 @@ def ER_WestlawNext(self, x, name='ER-WestlawNext'):
         x = utilities.MarcEditMakeFile(x)
         return x
 
-    def ER_WoltersKluwer_Cheetah(self, x, name='ER-WoltersKluwer-Cheetah'):

```

Comment:
Why did you remove this one entirely?

Reply:
I just forgot to add them back.
When we first spoke about the project we mentioned modularizing these functions so that's what I started doing but then we changed our mind so, I just kept working with everything as a whole and forgot to add it back it. I can do that now.